records_mover/url/base.py

```diff
@@ -48,6 +50,34 @@ def directories_in_directory(self) -> List['BaseDirectoryUrl']:
         "Return entries in this directory."
         raise NotImplementedError()
 
+    def empty(self) -> bool:
+        # TODO document
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker